spec/controllers/users_controller_spec.rb

```diff
@@ -128,6 +128,53 @@ def valid_session
     end
   end
 
+  describe '#feedbacks' do
+    let(:user) { create(:user) }
+    before do
+      set_current_user user
+    end
+
+    it 'should assign unsubmitted feedbacks belonging to the user' do
+      feedbacks = create_list(:feedback, 2, user: user)
+      get :feedbacks, {id: user}, valid_session
+      assigns(:feedbacks).should =~ feedbacks
+    end
+
+    it 'should not assign submitted feedbacks' do
+      unsubmitted_feedback = create(:feedback, user: user)
```

Comment:
Useless assignment to variable - `unsubmitted_feedback`. Did you mean `submitted_feedback`?